code/game/objects/items/toys.dm

```diff
@@ -178,6 +178,27 @@
 	icon = 'icons/obj/singularity.dmi'
 	icon_state = "singularity_s1"
 
+/obj/item/toy/spinningtoy/suicide_act(mob/living/carbon/human/user)
+	var/obj/item/bodypart/head/myhead = user.get_bodypart(BODY_ZONE_HEAD)
+	if(!myhead)
+		user.visible_message("<span class='suicide'>[user] tries consuming [src]... but [user.p_they()] [user.p_have()] no mouth!</span>") // and i must scream
+		return SHAME
+	user.visible_message("<span class='suicide'>[user] consumes [src]! It looks like [user.p_theyre()] trying to commit suicicide!</span>")
+	playsound(user, 'sound/items/eatfood.ogg', 50, TRUE)
+	user.adjust_nutrition(50) // mmmm delicious
+	sleep(3 SECONDS)
```

Comment:
don't use sleep, use timers

Reply:
Saw sleep on fedora suicide and thought doing it here was okay, how would i go about doing this?

Reply:
hold on i think i know what i can do

Reply:
aight, sleep is changed to a callback timer